src/Navigation/WPF/Navigation.WPF/Service/NavigationService.show.cs

```diff
@@ -299,5 +270,24 @@ protected async Task<TViewModel> InitializeViewModel<TViewModel>()
                 viewModel,
                 content);
         }
+
+
+
+        private static TView GetContentInstance<TView>(
+            INavigationRegistration registration)
+
+            where TView : FrameworkElement
+        {
+            if (registration.ResolveViewUsingDependencyInjection)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use the '?:' operator here.](https://app.codacy.com/gh/UltimateCodeMonkeys/CodeMonkeysMVVM/pullRequest?prid=5575746)